### ISSUE TITLE
Motion Path

### ISSUE THREAD

**Issue by pepelsbey** [web-standards-ru/dictionary#270]
[Путь движения?](https://drafts.fxtf.org/motion-1/)

**Comment by andrew--r** [comment]
«Траектория», кажется, отражает суть лучше, чем «путь»

**Comment by spacesmugglerpotato** [comment]
Да, у обьёктов в анимации это называется траекторией. Так что траектория движения.

**Comment by SelenIT** [comment]
Я тоже за траекторию.

**Comment by pepelsbey** [comment]
Так может вообще траектория, без движения? :)

**Comment by SelenIT** [comment]
Мне кажется, зависит от контекста. Во фразе наподобие «...и плавно перемещаем этот объект по траектории», конечно, ничего добавлять не надо. Но вот название _CSS-модуль траекторий_, как мне кажется, всё-таки просит какого-то уточнения (или это мое искаженное восприятие?).

**Comment by askd** [comment]
Траектория анимации

**Comment by marinintim** [comment]
Траектория движения — плеоназм, как по мне. (:
